scipy/spatial/transform/_rotation.pyx

```diff
@@ -1835,6 +2107,151 @@ cdef class Rotation:
         """
         return self._compute_euler(seq, degrees, 'from_quat')
 
+    @cython.embedsignature(True)
+    def as_davenport(self, axes, order, degrees=False):
+        """Represent as Davenport angles.
+
+        Any orientation can be expressed as a composition of 3 elementary
+        rotations.
+
+        For Euler angles, we have the following conditions:
+
+            - ``dot(axis1, axis2) = 0``
+            - ``dot(axis2, axis3) = 0``
+            - ``dot(axis1, axis3) = -1``, ``0`` or ``1``
+
+        For Davenport angles, the third relation is relaxed [1]_:
+
+            - ``dot(axis1, axis3)`` can be any value between -1 and 1.
+
+        A slightly modified version of the algorithm from [2]_ has been used to
+        calculate Davenport angles for the rotation about a given sequence of
+        axes.
+
+        Davenport angles, just like Euler angles, suffer from the problem of
+        gimbal lock [3]_, where the representation loses a degree of freedom
+        and it is not possible to determine the first and third angles
+        uniquely. In this case, a warning is raised, and the third angle is set
+        to zero. Note however that the returned angles still represent the
+        correct rotation.
+
+        Parameters
+        ----------
+        axes : array_like
+            Specifies sequence of axes for rotations. Must be an array_like of
+            shape (3, ) or (3, [1 or 2 or 3]), where each axes[i, :] is the ith
+            axis. If more than one axis is given, then the second axis must be
+            orthogonal to both the first and third axes.
+        order : string
+            If it belongs to the set {'e', 'extrinsic'}, the sequence will be 
+            extrinsic. If if belongs to the set {'i', 'intrinsic'}, sequence 
+            will be treated as intrinsic.
+        degrees : boolean, optional
+            Returned angles are in degrees if this flag is True, else they are
+            in radians. Default is False.
+
+        Returns
+        -------
+        angles : ndarray, shape (3,) or (N, 3)
+            Shape depends on shape of inputs used to initialize object.
+            The returned angles are in the range:
+
+            - First angle belongs to [-180, 180] degrees (both inclusive)
+            - Third angle belongs to [-180, 180] degrees (both inclusive)
+
+            - Second angle belongs to a set of size 180 degrees, 
+            given by: ``[-abs(lambda), 180 - abs(lambda)]``, where ``lambda``
```

Comment:
Or another variant: perhaps you need to align "given by" and "is the" to the "Second" in the first line?

Reply:
Ah, probably this one! I'll try it, thanks!

Reply:
Solved it!